vagrant/provisioning/roles/MongoDB/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+- name: configure Mongodb 4.0 repository
+  become: yes
+  copy:
+    dest: /etc/yum.repos.d/mongodb-org.repo
+    force: no
+    content: |
+      [mongodb-org-4.0]
+      name=MongoDB Repository
+      baseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/4.0/x86_64/
+      gpgcheck=1
+      enabled=1
+      gpgkey=https://www.mongodb.org/static/pgp/server-4.0.asc
+
+- name: install Mongodb
+  become: yes
+  yum:
+    state: installed
+    disable_gpg_check: yes
+    name:
+      - mongodb-org
+  register: mongodb_install
+
+- name: Mongodb folders
+  become: yes
+  file:
+    path: "{{ item.name }}"
+    state: directory
+    group: mongod
+    owner: mongod
+    setype: "{{ item.setype }}"
+  loop:
+    - name: "{{ mongodb_data_folder }}"
+    #SELinux context on the new dbpath
+      setype: "mongod_var_lib_t"
+    - name: "{{ mongodb_log_folder }}"
+    #SELinux context on the new logpath
+      setype: "mongod_log_t"
+  register: mongodb_folder_structure
+
+- name: Change path to the data folder
+  become: yes
+  replace: 
+    path: /tmp/mongod.conf
```

Comment:
Why is this file in /tmp ? I cannot see it moved somewhere later? The contents in /tmp directory are deleted by the system after some time automatically.

Reply:
Sorry about that, I was performing some tests locally and forgot to change for production. Will do it asap.